Sandbox/src/Game/Interface/MenuTabsInterface.cpp

```diff
@@ -0,0 +1,192 @@
+#include "MenuTabsInterface.h"
+
+MenuTabsInterface::MenuTabsInterface(Player& player)
+	: mPlayer(player)
+{
+	mCommand = "";
+	mWaitingForInteraction = false;
+
+	AddButton("Magic");
+	AddButton("Prayer");
+	AddButton("Gear");
+	AddButton("Inventory");
+	AddButton("Stats");
+	AddButton("Combat");
+
+	mTabs["Inventory"] = new InventoryInterface(mPlayer.Inventory());
+	mTabs["Inventory"]->Parent(this);
+	mTabs["Inventory"]->Pos(Square::Vector2(Square::Graphics::SCREEN_WIDTH - 208, Square::Graphics::SCREEN_HEIGHT - 256 - mButtons["Inventory"]->Height()));
```

Comment:
I can see this as a re-usable helper function in case we ever wanna change the dimensions
`int GetTabPosition(int offsetHeight)`